working/augmentation-libraries/feature-specification.md

```diff
@@ -752,8 +751,8 @@ augment class C {
 ```
 
 This code is fine. Code in C in the main library can refer to members added in
```

Comment:
"in C" -> "in the class `C`"

(Code quote source names, and make it clear that there is no C (the language) code here.)